cpp/open3d/visualization/visualizer/GuiVisualizer.cpp

```diff
@@ -982,13 +1035,14 @@ void GuiVisualizer::LoadGeometry(const std::string &path) {
             if (success) {
                 utility::LogInfo("Successfully read {}", path.c_str());
                 UpdateProgress(ioProgressAmount);
-                if (!cloud->HasNormals()) {
+                if (!cloud->HasNormals() && !cloud->HasColors()) {
```

Comment:
[Discussing] May we make this optional too, with a console warning, for example: `The point cloud {file_name} file has neither normals nor colors, Please use 'Estimate PCD Normals' to add normals to the point cloud for better visualization`?

Reply:
ignore this comment.

Reply:
Discussed at meeting.